src/web/src/components/Posts/Post.tsx

```diff
@@ -349,7 +349,7 @@ const PostComponent = ({ postUrl, currentPost, totalPosts }: Props) => {
             <div className={classes.publishedDateContainer}>
               <h1 className={classes.published}>
                 <a href={post.url} rel="bookmark" className={classes.link}>
-                  {`${formatPublishedDate(post.updated)}`}
+                  <time dateTime={post.updated}>{`${formatPublishedDate(post.updated)}`}</time>
```

Comment:
Bless you.  I [wrote the original implementation of `<time>` in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=629801), and I love seeing it used in the wild!

Reply:
Wow. That's crazy.